openbb_platform/providers/yfinance/openbb_yfinance/models/crypto_historical.py

```diff
@@ -72,12 +72,21 @@ def extract_data(
         **kwargs: Any,
     ) -> List[Dict]:
         """Return the raw data from the Yahoo Finance endpoint."""
-        if "-" not in query.symbol:
-            position = len(query.symbol) - 3
-            query.symbol = query.symbol[:position] + "-" + query.symbol[position:]
+
+        tickers = query.symbol.split(",") if "," in query.symbol else [query.symbol]
```

Comment:
split method will actually return a list if its not in there so you can drop the conditional